tests/test_report_manager.py

```diff
@@ -389,7 +361,7 @@ def _test_summary_counts(self, add_table_fn, add_plot_fn,
         avg_gpu_metrics = {0: {"gpu_used_memory": 6000, "gpu_utilization": 60}}
         for i in range(10):
             p99 = 20 + i
-            throughput = 100 - 10 * i if default_within_top else 100 + 10 * i
+            throughput = 100 - 10 * i if default_within_top else 200 + 10 * i
```

Comment:
This is beyond magic numbers. This is a whole spellbook 😛.  
Can we add a comment to explain this formula?

Reply:
done